docs/content/operations/rule-configuration.md

```diff
@@ -126,6 +129,65 @@ Period drop rules are of the form:
 
 The interval of a segment will be compared against the specified period. The period is from some time in the past to the current time. The rule matches if the period contains the interval.
 
+Broadcast Rules
+---------------
+
+Broadcast rules indicate how segments of different data sources should be co-located in historical nodes. 
+Once a broadcast rule is configured for a data source, all segments of the data source are broadcasted to the servers holding _any segments_ of the co-located data sources.
+
+### Forever Broadcast Rule
+
+Forever broadcast rules are of the form:
+
+```json
+{
+  "type" : "broadcastToDataSourcesForever",
```

Comment:
I think this should just be called `broadcastForever`, since the `dataSources` part is optional. And similar comment for the other kinds of broadcast rules.